1.74/api/index.html

```diff
@@ -232,4 +230,4 @@ <h1>Indices and tables<a class="headerlink" href="#indices-and-tables" title="Pe
    
 
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
If you're adding this, then we need an opening ``<html>`` tag too. Maybe just go for the minimal change?

Looks like this bug in Sphinx has been fixed, our 1.78 docs has ``<html class="writer-html5" lang="en" >`` near the start.

Reply:
I guess this closing `<\html>` was automatically added because the one which was there before had no newline.

Reply:
So, I did not add it, it was replaced.

Reply:
Ah - GitHub is trying to tell me there was no trailing line symbol. I see the opening ``<html>`` now hidden in some magic comma if statements for Internet Explorer.

Sorry, false alarm.